packages/dca-frontend/src/components/wallet.tsx

```diff
@@ -18,37 +18,39 @@ const formatAddress = (address: string | undefined) => {
 };
 
 export const Wallet: React.FC = () => {
-  const { chain, provider, wethContract } = useChain();
+  const { chain, provider, usdcContract, wbtcContract } = useChain();
   const [ethBalance, setEthBalance] = useState<string>('0');
-  const [wethBalance, setWethBalance] = useState<string>('0');
+  const [usdcBalance, setUsdcBalance] = useState<string>('0');
+  const [wbtcBalance, setWbtcBalance] = useState<string>('0');
   const [isLoadingBalance, setIsLoadingBalance] = useState<boolean>(false);
   const [error, setError] = useState<string | null>(null);
   const { authInfo, logOut } = useJwtContext();
 
-  // Function to fetch PKP wethBalanceWei directly using ethers.js
+  // Function to fetch PKP balances
   const fetchPkpBalance = useCallback(async () => {
     if (!authInfo?.pkp.ethAddress) return;
 
     try {
       setIsLoadingBalance(true);
       setError(null);
 
-      const [ethBalanceWei, wethBalanceWei] = await Promise.all([
+      const [ethBalanceWei, usdcBalance, wbtcBalanceWei] = await Promise.all([
         provider.getBalance(authInfo?.pkp.ethAddress),
-        wethContract.balanceOf(authInfo?.pkp.ethAddress),
+        usdcContract.balanceOf(authInfo?.pkp.ethAddress),
+        wbtcContract.balanceOf(authInfo?.pkp.ethAddress),
       ]);
 
-      // Both have 18 decimal places
-      setEthBalance(ethers.utils.formatEther(ethBalanceWei));
-      setWethBalance(ethers.utils.formatEther(wethBalanceWei));
+      setEthBalance(ethers.utils.formatUnits(ethBalanceWei, 18));
+      setUsdcBalance(ethers.utils.formatUnits(usdcBalance, 6));
+      setWbtcBalance(ethers.utils.formatUnits(wbtcBalanceWei, 18));
```

Comment:
wBtc has [8 decimals](https://basescan.org/token/0x0555E30da8f98308EdB960aa94C0Db47230d2B9c#readContract#F10) not 18
```suggestion
      setWbtcBalance(ethers.utils.formatUnits(wbtcBalanceWei, 8));
```

Reply:
Does this new change make any difference in our DCA logic anywhere?

Reply:
Fixed. No, `formatUnits` is only used for UI things